docs/get_started/first_isvc.md

```diff
@@ -186,6 +186,16 @@ Depending on your setup, use one of the following commands to curl the `Inferenc
     curl -v -H "Content-Type: application/json" http://sklearn-iris.kserve-test/v1/models/sklearn-iris:predict -d @./iris-input.json
     ```
 
+=== "Port forwarded service"
+
+    If you have done `Port Forward` above during testing, you can extract the inference
+    server URL from the status and make a curl request to it.
+
+    ```bash
+    SERVICE_HOSTNAME=$(kubectl get inferenceservice sklearn-iris -n kserve-test -o jsonpath='{.status.url}' | cut -d "/" -f 3)
+    curl -v -H "Host: ${SERVICE_HOSTNAME}" -H "Content-Type: application/json" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/sklearn-iris:predict" -d @./iris-input.json
```

Comment:
Maybe instead adding a new section, what about add the command curl command after https://github.com/kserve/website/pull/412/files#diff-2066f8cd5b8985c4a57b1da95f0b1da8277507031b72a35759bd066b47d11ee3R121

btw, is think there is no need to pass the SERVICE_HOSTNAME

Reply:
I updated the PR to add the curl command where you suggested. I think I had to modify the SERVICE_HOSTNAME section to work correctly.